src/components/Expand/index.tsx

```diff
@@ -22,22 +22,25 @@ export default function Expand({
   button,
   children,
   testId,
+  isOpen,
```

Comment:
You can name the prop `open` and still have a better naming internally:

```suggestion
  open: isOpen,
```

Reply:
isOpen feels more canonical  - booleans usually start with `is` or `should`